From 2af564865343fa4b384fdb6c61138c07b8601cba Mon Sep 17 00:00:00 2001
From: Ayyappa Chandolu <Ayyappa.Chandolu@amd.com>
Date: Fri, 2 Mar 2018 11:25:17 +0530
Subject: [PATCH 110/131] pinctrl: eMMC and PinCtrl is sharing the interrupt no
 7 for Dibber. So PinCtrl must register the interrupt hadler with SHARED
 flags. BUGID : EMBSWDEV-4739

Signed-off-by: Ayyappa Chandolu <Ayyappa.Chandolu@amd.com>
Signed-off-by: Sudheesh Mavila <sudheesh.mavila@amd.com>
---
 drivers/pinctrl/pinctrl-amd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pinctrl/pinctrl-amd.c b/drivers/pinctrl/pinctrl-amd.c
index b78f42a..f0e1f7c 100644
--- a/drivers/pinctrl/pinctrl-amd.c
+++ b/drivers/pinctrl/pinctrl-amd.c
@@ -896,7 +896,7 @@ static int amd_gpio_probe(struct platform_device *pdev)
 		goto out2;
 	}
 
-	ret = devm_request_irq(&pdev->dev, irq_base, amd_gpio_irq_handler, 0,
+	ret = devm_request_irq(&pdev->dev, irq_base, amd_gpio_irq_handler, IRQF_SHARED,
 			       KBUILD_MODNAME, gpio_dev);
 	if (ret)
 		goto out2;
-- 
2.7.4

